packages/backend/routes/roles.ts

```diff
@@ -0,0 +1,19 @@
+import express, { Request, Response, NextFunction, Router } from 'express';
+import User from '../models/user/user';
+import RoleController from '../controllers/role';
+
+const router: Router = express.Router();
+
+/* GET all users. */
+router.get(
+  '/my_roles',
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
If you don't need the `next` var, you might be able to drop the `eslint-disable` if you remove the param

Reply:
oh sick. good call.